packages/hadron-type-checker/src/type-checker.ts

```diff
@@ -293,19 +333,15 @@ const INT64_CHECK = new Int64Check();
 
 /**
  * Gets the BSON type for a JS number.
- *
- * @param {Number} number - The number.
- *
- * @returns {String} The BSON type.
  */
-const numberToBsonType = (number) => {
-  var string = toString(number);
+const numberToBsonType = (number: number): TypeCastTypes => {
+  const string = toString(number);
   if (INT32_CHECK.test(string)) {
-    return INT_32;
+    return INT_32 as TypeCastTypes;
```

Comment:
[super-nit]: do we need these (here and elsewhere)?
```suggestion
    return INT_32;
```

Reply:
Nope! Nice catch, removed.